src/message_interface/response.rs

```diff
@@ -56,3 +58,30 @@ pub enum Response {
     /// All went fine.
     Ok(()),
 }
+
+// Custom Debug implementation to not log secrets
+impl Debug for Response {
+    fn fmt(&self, f: &mut Formatter<'_>) -> Result {
+        match self {
+            Response::Account(account) => write!(f, "Account({:?})", account),
+            Response::Accounts(accounts) => write!(f, "Accounts({:?})", accounts),
+            Response::Addresses(addresses) => write!(f, "Addresses({:?})", addresses),
+            Response::AddressesWithUnspentOutputs(addresses) => {
+                write!(f, "AddressesWithUnspentOutputs({:?})", addresses)
+            }
+            Response::OutputIds(output_ids) => write!(f, "OutputIds({:?})", output_ids),
+            Response::Output(output) => write!(f, "OutputData({:?})", output),
+            Response::Outputs(outputs) => write!(f, "Outputs{:?}", outputs),
+            Response::Transactions(transactions) => write!(f, "Transactions({:?})", transactions),
+            Response::GeneratedAddress(addresses) => write!(f, "GeneratedAddress({:?})", addresses),
+            Response::Balance(balance) => write!(f, "Balance({:?})", balance),
+            Response::SentTransfer(transfer) => write!(f, "SentTransfer({:?})", transfer),
+            Response::SentTransfers(transfers) => write!(f, "SentTransfers({:?})", transfers),
+            Response::Error(error) => write!(f, "Error({:?})", error),
+            Response::Panic(panic_msg) => write!(f, "Panic({:?})", panic_msg),
+            Response::GeneratedMnemonic(_) => write!(f, "GeneratedMnemonic()"),
```

Comment:
I'd be a bit more explicit about what's happening here, maybe something like `GeneratedMnemonic(<omitted>)`

Reply:
Or `...`

Reply:
Yeah but even just that feels like something the std could have done for some reason